hack/test-e2e.sh

```diff
@@ -77,9 +77,9 @@ else
     oc wait pod/elasticsearch-operator-e2e-test -n ${TEST_NAMESPACE} --for=condition=PodScheduled -o yaml --timeout=300s
 
     oc wait pod/elasticsearch-operator-e2e-test -n ${TEST_NAMESPACE} --for=condition=Ready --timeout=300s || oc logs elasticsearch-operator-e2e-test -n ${TEST_NAMESPACE}
-    oc get pods -n ${TEST_NAMESPACE} || :
+    oc get pods -n ${TEST_NAMESPACE}
 
-    oc logs -f elasticsearch-operator-e2e-test -n ${TEST_NAMESPACE}
+    #oc logs -f elasticsearch-operator-e2e-test -n ${TEST_NAMESPACE}
```

Comment:
we need to have some way to get the logs from pod/elasticsearch-operator-e2e-test

Reply:
true, but we don't even see output from the `oc get pods -n ${TEST_NAMESPACE}` above